functions/mqhandlers.go

```diff
@@ -373,9 +373,7 @@ func HostUpdate(client mqtt.Client, msg mqtt.Message) {
 			slog.Error("failed to response with ACK to server", "server", serverName, "error", err)
 		}
 		setSubscriptions(client, &nodeCfg)
-		if server.ManageDNS {
-			setDNSSubscriptions(client, &nodeCfg)
-		}
+		setDNSSubscriptions(client, &nodeCfg)
```

Comment:
why do we need to setup subscription here again?

Reply:
Yes,   I think so.  In one of the tests, the subscription missed, that's why it's added here. 
setSubscriptions is called here as well.